src/gwt/src/org/rstudio/studio/client/workbench/views/environment/dataimport/DataImport.java

```diff
@@ -33,28 +44,58 @@
    private static DataImportUiBinder uiBinder = GWT
          .create(DataImportUiBinder.class);
    
+   private final SourceServerOperations server_;
+   private final int maxRows_ = 200;
+   
    interface DataImportUiBinder extends UiBinder<Widget, DataImport>
    {
    }
 
-   public DataImport()
+   public DataImport(SourceServerOperations server)
    {
       initWidget(uiBinder.createAndBindUi(this));
       
+      server_ = server;
+      
       Size size = DomMetrics.adjustedElementSizeToDefaultMax();
       setSize(size.width + "px", size.height + "px");
+      
+      int previewHeight = size.height - 60;
```

Comment:
Yes, @jcheng5 pointed out we could use the "new" flexbox! @jmcphers discussed this as well and pointed to make sure this works on IE10. Gave it a shot in IE10, IE11 and Edge all looking good. Same for Safari. I'll test this a bit more before I wrap up this feature for sure.